pom.xml

```diff
@@ -863,6 +863,17 @@
         <optional>true</optional>
       </dependency>
 
+      <!--
+        Completely optional integration with Bouncy Castle TLS.
```

Comment:
maybe add some comment that this is needed for OSGI so we not remove it later again because we don't realize its important

Reply:
make sense, I wasn't sure how much detail to add.
follow-up patch contains pretty much a brain dump -- I guess it is better to over-communicate here :)